File: trail1/별표 출력하기 12/print-star-12.py
N = int(input())
board = [[' ' for _ in range(N)] for _ in range(N)]
for i in range(N):
    for j in range(N):
        if i == 0:
            board[i][j] = '*'
        if j % 2 == 1 and j >= i:
            board[i][j] = '*'

for i in range(N):
    for j in range(N):
        print(board[i][j], end=' ')
    print()

File: trail1/별표 출력하기 12/print-star-12.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        char[,] board = new char[N, N];
        for (int i=0; i<N; i++) {
            for (int j=0; j<N; j++) {
                if (i==0 || (j%2==1 && j>=i)) {board[i,j] = '*';}
                else {board[i,j] = ' ';}
            }
        }
        for (int i=0; i<N; i++) {
            for (int j=0; j<N; j++) {
                Console.Write(board[i,j]+ " ");
            }
            Console.WriteLine();
        }
    }
}
